<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertDialog</class>
 <widget class="QDialog" name="insertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="namelabel">
       <property name="text">
        <string>书名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="authorlabel">
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="authorEdit">
       <property name="placeholderText">
        <string>使用英文逗号分隔多位作者</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="yearlabel">
       <property name="text">
        <string>出版年</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yearEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="publisherlabel">
       <property name="text">
        <string>出版社</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="publisherEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="loclabel">
       <property name="text">
        <string>出版位置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="locEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="pageslabel">
       <property name="text">
        <string>页数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pagesEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>insertDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>insertDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkText()</slot>
 </slots>
</ui>
